tests/model-proxy.spec.js

```diff
@@ -142,11 +142,11 @@ describe('ModelProxy', function () {
     sinon.assert.calledOnce(spyB)
   })
 
-  it('isPending is true when the promise has not been resolved or rejected')
-  it('isResolved is true when the promise has been resolved')
-  it('isResolved is false when the promise has been rejected')
-  it('isRejected is true when the promise has been rejected')
-  it('isRejected is false when the promise has been resolved')
+  it('isPending is true when the promise has not been resolved or rejected', () => {})
+  it('isResolved is true when the promise has been resolved', () => {})
+  it('isResolved is false when the promise has been rejected', () => {})
+  it('isRejected is true when the promise has been rejected', () => {})
+  it('isRejected is false when the promise has been resolved', () => {})
```

Comment:
Maybe mark these tests as `skip` instead of empty?

Reply:
@al-the-x I am all for skipping these tests, but I am trying to preserve existing behavior while just performing the upgrade. We need to have a function as of [Jest 23](https://jestjs.io/blog/2018/05/29/jest-23-blazing-fast-delightful-testing#breaking-changes)

Reply:
We can omit the test function if we use `it.todo`, which was previous behavior. That used to work with `it.skip`, too.

Reply:
Agreed; 'existing behavior' is that no one knew these tests did nothing until this Jest upgrade. Without the added visibility of `todo` or `skip`, it is likely they will continue on silently doing nothing.